rudder-web/src/main/scala/com/normation/rudder/web/components/RuleEditForm.scala

```diff
@@ -1440,7 +1455,12 @@ class RuleEditForm(
                "bLengthChange": true,
                "sPaginationType": "full_numbers",
                "bStateSave": true,
-               "sCookiePrefix": "Rudder_DataTables_",
+                    "fnStateSave": function (oSettings, oData) {
+                      localStorage.setItem( 'DataTables_${gridId}', JSON.stringify(oData) );
+                    },
```

Comment:
I don't think we want one option by grid. The used will expect to have AT MOST one settings for each grid he sees - by for him, all "details" table are the same. 

In the same idea, all node grid details are the same, etc.